Makefile

```diff
@@ -75,7 +75,11 @@ docker: certs build-in-docker pause-container-release cni-plugins .out-stamp
 # 'RELEASE' mode
 # TODO: make this idempotent
 docker-release: pause-container-release cni-plugins .out-stamp
+ifeq ($(TARGET_OS),linux)
 	@docker build -f scripts/dockerfiles/Dockerfile.cleanbuild -t "amazon/amazon-ecs-agent-cleanbuild:make" .
+else ifeq ($(TARGET_OS),windows)
+	@docker build -f scripts/dockerfiles/Dockerfile.cleanbuildWindows -t "amazon/amazon-ecs-agent-cleanbuild:make" .
+endif
```

Comment:
Will this break for mac os?

Reply:
Good catch! I was only thinking about `codebuild` target use case. I will update it